recipe/meta.yaml

```diff
@@ -1,117 +1,123 @@
 {% set name = "sage" %}
-{% set version = "8.1" %}
+{% set version = "8.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 build:
-  number: 1
+  number: 0
   skip: true  # [win]
   skip: true  # [py!=27]
 
 requirements:
-  build:
+  host:
     - python
   run:
     # We require all dependencies with at least that version that upstream uses
     # in their SPKGs. For packages that (seem to) use semantic versioning and
     # are at least 1.0.0, we limit the version to the next major release.
-    - alabaster >=0.7.8
+    - alabaster >=0.7.10
     - appnope >=0.1.0  # [osx]
-    - arb >=2.11.1,<3
+    - arb >=2.12.0,<3
+    - babel >=2.3.4,<3
+    - backports.functools_lru_cache >=1.4,<2
     - backports.shutil_get_terminal_size >=1.0.0,<2
     - backports_abc >=0.5
     - bdw-gc >=7.2f,<8
+    - bleach >=2.1.3,<3
     - boost-cpp >=1.61.0,<2
     - brial >=1.0.1,<2
     - cddlib >=0.94g
-    - certifi >=2016.2.28
+    - certifi >=2017.11.5
     - cliquer >=1.21,<2
     - configparser >=3.5.0,<4
     - cvxopt >=1.1.8,<2
     - cycler >=0.10.0
-    - cysignals >=1.6.5,<2
+    - cypari2 >=1.1.4,<2
+    - cysignals >=1.7.0,<2
+    - cython >=0.28.1
     - decorator >=4.0.6,<5
-    - docutils >=0.13.1
+    - docutils >=0.14
     - ecl >=16.1.2,<17
     - eclib >=20171002
     - ecm >=7.0.4,<8
-    - entrypoints >=0.2.2
-    - fflas-ffpack >=2.2.2,<2.3.0
+    - entrypoints >=0.2.3
+    - fflas-ffpack >=2.3.2,<2.4.0
     - flintqs >=1.0,<2
     - fplll >=5.2.0,<6
     - fpylll >=0.3.0dev
-    - freetype >=2.6.3,<3
+    # currently, conda-forge's matplotlib builds against freetype=2.6, so we
+    # can not use the version 2.8 that SageMath proposes.
+    - freetype >=2.6,<3
```

Comment:
Instead of downgrading freetype, can you upgrade the matplotlib version?

Reply:
Sure. Let's do this as well.

Reply:
Actually the latest matplotlib does use 2.8.*. Let's see why sage wanted to pull in a later version.

Reply:
Oh, now I understood what you meant. Upgrade matplotlib here. I thought you meant upgrade matplotlib so it uses the latest freetype.